packages/application-extension/src/index.tsx

```diff
@@ -385,24 +416,28 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         label: trans.__('Simple Interface'),
         isToggled: () => labShell.mode === 'single-document',
         execute: () => {
+          let ariaLiveRegion = document.getElementById('commands-aria-live');
           const args =
             labShell.mode === 'multiple-document'
               ? { mode: 'single-document' }
               : { mode: 'multiple-document' };
+          ariaLiveRegion!.append(trans.__('Simple Interface'));
           return commands.execute(CommandIDs.setMode, args);
         }
       });
 
       commands.addCommand(CommandIDs.resetLayout, {
         label: trans.__('Reset Default Layout'),
         execute: () => {
+          let ariaLiveRegion = document.getElementById('commands-aria-live');
           // Turn off presentation mode
           if (labShell.presentationMode) {
             commands
               .execute(CommandIDs.togglePresentationMode)
               .catch(reason => {
                 console.error('Failed to undo presentation mode.', reason);
               });
+            ariaLiveRegion!.append(trans.__('Turn off presentation mode'));
```

Comment:
Since it happens after the action was performed, should it be more like "Presentation mode turned off"?

Should it be in `.then()` to only show when action succeeded?

Should there be an alternative next to `console.error` in `catch` when it failed?